backend/climateconnect_api/views/user_views.py

```diff
@@ -233,14 +233,16 @@ def post(self, request):
         if 'image' in request.data:
             user_profile.image = get_image_from_data_url(request.data['image'])[0]
         if 'background_image' in request.data:
-            user_profile.background_image = get_image_from_data_url(request.data['background_image'], True, 1280)[0]
-        if 'country' in request.data:
-            user_profile.country = request.data['country']
-
-        if 'state' in request.data:
-            user_profile.state = request.data['state']
-        if 'city' in request.data:
-            user_profile.city = request.data['city']
+            user_profile.background_image = get_image_from_data_url(
+                request.data['background_image'], True, 1280
+            )[0]
+
+        if 'location' in request.data:
+            geo_location = get_geo_location(request.data['location'])
+            user_profile.location = geo_location['location']
+            user_profile.latitude = geo_location['latitude']
+            user_profile.longitude = geo_location['longitude']
+
```

Comment:
I think because of this you are getting wrong location? Don't we have to change the add a new column?

Reply:
You're right, we need to change this. But I'm only testing with edit project atm, so this can't be the problem

Reply:
But you're right, we have to remove this